Dan.Common/Util/KeyedLock.cs

```diff
@@ -7,49 +8,35 @@ public class KeyedLock<TKey> where TKey : notnull
 {
     private readonly AsyncKeyedLocker<TKey> _asyncKeyedLocker;
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Hey [: Is there a measured reason to add the AggressiveInlining attribute to the methods? In general I prefer to avoid adding MethodImplOptions unless I know that the compiler or jitter is doing something that noticably messes with performance

Reply:
I've benchmarked the calls in my methods and found a benefit to using inlining. Since this method is making a call to the inlined method in the library, then it stands to reason there will indeed be a benefit, though in this very particular line of code with regards to inlining the constructor there isn't going to be any discernible difference.